tests/utils/siteBuilder.js

```diff
@@ -1,10 +1,15 @@
 const { toToml } = require('tomlify-j0.4')
-const fs = require('fs-extra')
 const path = require('path')
 const { v4: uuidv4 } = require('uuid')
 const tempDirectory = require('temp-dir')
 const os = require('os')
 
+const fs = require('../../src/lib/fs')
+
+const ensureDir = async path => {
+  return fs.mkdirRecursiveAsync(path)
+}
```

Comment:
Looking at this again now, this function probably can be removed and `fs.mkdirRecursiveAsync` used directly.

Reply:
Good point